### PR TITLE
Issue 26: Pass request for PDF.js extension install to the extension updater proxy

### LINKED ISSUE #26: Pass requests for PDF.js extension install to extension updater proxy

### PR DESCRIPTION
**Pull request by jumde**
fix #26

### CODE REVIEW

**Comment by bbondy** [comment]
Please make sure that a fresh install still gets rid of the browser action button. I think it still does and it's done dynamically on the client side. Just as a spot check.

File: controller/controller.go [hunk 1]
```diff
@@ -20,6 +20,10 @@ import (
 	"time"
 )
 
+// PDFjsExtensionID will be used to add an exception to pass the request for
+// PDF viewer extension install from chrome web store to the extension updater proxy
+var PDFjsExtensionID = "oemmndcbldboiebfnladdacbdfmadadm"
```
Comment (diracdeltas) [inline controller/controller.go]: minor: should probably be `PDFJS` not `PDFjs` for consistency

Comment (jumde) [reply controller/controller.go]: resolved

**Comment by diracdeltas** [comment]
sec review passes

**Pull request merged**